test/integration/smoke-testing/test-login-failures.js

```diff
@@ -31,17 +29,17 @@ test('Trying to sign in with no password using scratchr2 navbar', t => {
     var nonsenseusername = Math.random().toString(36)
         .replace(/[^a-z]+/g, '')
         .substr(0, 5);
-    clickText('Sign in')
-        .then(() => findByXpath('//input[@id="login_dropdown_username"]'))
+    clickCss('.dropdown-toggle')
+        .then(() => findByCss('form#login input#login_dropdown_username'))
         .then((element) => element.sendKeys(nonsenseusername))
-        .then(() => clickButton('Sign in'))
-        .then(() => driver.wait(until
-            .elementLocated(By.xpath('//form[@id="login"]/button[@type="submit"]'))))
-        .then(() => driver.wait(until
-            .elementLocated(By.xpath('//form[@id="login"]/div[@class="error"]'))))
-        .then(() => findByXpath('//form/div[@class="error"]'))
+        .then(() => clickCss('form#login button'))
+        .then(() => findByCss('form#login .error'))
```

Comment:
Same comment here re: ids, although I think here it's probably ok since all you're doing is converting from xpath to css selectors.  But these can probably be simplified to `#login button` and `#login .error` respectively.